keps/262-ConfigurableFailurePolicy/kep.yaml

```diff
@@ -0,0 +1,29 @@
+title: Configurable Failure Policy
+kep-number: 262
+authors:
+  - "@danielvegamyhre"
+status: provisional
+creation-date: 2023-08-01
+reviewers:
+  - "@kannon92"
+  - "@vsoch"
+  - "@mimowo"
+approvers:
+  - "@ahg"
+
+see-also:
+  - "NA"
```

Comment:
Point to PodFailurePolicy and your KEP to add the reason to it

Reply:
Added